practica2023/src/main/java/com/endava/practica2023/controller/OrderController.java

```diff
@@ -24,4 +30,24 @@ public OrderController(OrderService orderService) {
     public List<Order> getOrderList(@RequestParam Integer id){
         return orderService.getOrderById(id);
     }
+
+    @Autowired
+    private TicketCategoryRepo ticketCategoryRepo;
+
+    @Autowired
+    private OrderRepo orderRepo;
+    @PostMapping("/POST_orders")
+    public UsefulInfo postOrder(@RequestBody NewOrder newOrder){
+        LocalDateTime localDateTime= LocalDateTime.now();
```

Comment:
Controller layer needs to communicate with the service, not the repo. Also all the business logic in this method should be moved to the service. The controller should only call the service method and return the result.